src/sentry/static/sentry/app/components/list/utils.tsx

```diff
@@ -24,7 +24,15 @@ const bulletStyle = (theme: Theme) => css`
   }
 `;
 
-const numericStyle = (theme: Theme, isSolid = false) => css`
+type Options = {
+  isSolid?: boolean;
+  initialCounterValue?: number;
+};
+
+const numericStyle = (
+  theme: Theme,
+  {isSolid = false, initialCounterValue = 0}: Options
```

Comment:
should it default to 0 or 1?

Reply:
@EvanPurkhiser Good question. Starting at `0` means the first step you see is actually `1`.

Reply:
maybe we can do some math on this value then so it matches what you see